thoth/glyph/cli.py

```diff
@@ -87,5 +88,23 @@ def generate(output: str) -> None:
             output_file.write("some output generated to file\n")
         _LOGGER.info("Generated output stored in %r", output)
 
+@cli.command()
+@click.option(
+    "--message",
+    "-m",
+    type=str,
+    default="-",
+    help="Commit message to be classified",
+)
+def classify(message: str) -> None:
+    """Generate CHANGELOG entries from the current Git project."""
+    _LOGGER.info("Hello, glyph!")
+    if message == "-":
+        click.echo("Please enter message")
+    else:
+        classifier = load_model("../../data/model_commits_v2_quant.bin") 
```

Comment:
Can the model become a parameter and default to the one provided? We could install it from Ceph. The data will need to go to setup.py so they are shipped with the package:

https://docs.python.org/3/distutils/setupscript.html#installing-package-data